app/src/main/java/org/simple/clinic/instantsearch/InstantSearchScreen.kt

```diff
@@ -287,7 +287,13 @@ class InstantSearchScreen :
         .textChanges()
         .skipInitialValue()
         .debounce(500, TimeUnit.MILLISECONDS)
-        .map { SearchQueryChanged(it.toString()) }
+        .map { searchQuery ->
+          val trimmedSearchQuery = searchQuery
+              .trim()
+              .toString()
```

Comment:
Do we need to convert this to toString? Isn't searchQuery a string already?

Reply:
It's a charsequence, so we need to convert it to string in the end.